app/pages/admin/project-status.jsx

```diff
@@ -73,13 +82,43 @@ class ProjectStatus extends Component {
       .catch(error => this.setState({ error }));
   }
 
+  handleDialogCancel() {
+    this.setState({ dialogIsOpen: false });
+  }
+
+  handleDialogSuccess(workflow) {
+    const promises = [
+      workflow.update({ active: false }).save(),
+      this.state.project.update({ 'configuration.default_workflow': undefined }).save()
+    ];
+
+    Promise.all(promises)
```

Comment:
I'm rethinking the use of `Promise.all()` since the possibility of a failure condition on the project PUT. We don't want the workflow changed if updating the default workflow fails. It made sense initially to have a single `then()` which `Promise.all()` allows us, but I'm thinking this will be safer:
```
this.state.project.update({ 'configuration.default_workflow': undefined }).save()
  .then(() -> {
     workflow.update({ active: false }).save()
      .then(this.getProjectAndWorkflows() // Only get project and workflows after successful project and workflow PUT
     ).catch(// its own catch);
  }).catch(// its own catch)
  .then(() => {
    this.setState({ dialogIsOpen: false }); 
  })
```
This is just a rough idea without testing, so it might need tweaks.